README.md

```diff
@@ -427,10 +427,94 @@ Called when the bounds in the calendar's canvas change. Use it for example to lo
 
 ## itemRenderer
 
-React component that will be used to render the item content. Will be
-passed the `item` and the `selected` as a props.
-
-Using complex components may result in performance problems.
+Render prop function used to render a customized item. The function provides multiple paramerters that can be used to render each item.
+
+Paramters provided to the function has two types: context params which have the state of the item and timeline, and prop getters functions
+
+
+#### Render props params
+
+##### context
+
+- `item` has the item we passed as a prop to the calendar.
+
+- `timelineContext`
+
+| property               | type              | description                                                                                 |
+| ---------------------- | ----------------- | ------------------------------------------------------------------------------------------- |
+| `timelineWidth` | `number`    | returns the full width of the timeline.            |
+| `visibleTimeStart`        | `number`    | returns the exact start of view port of the calender                 |
+| `visibleTimeEnd`         | `number`    | returns the exact end of view port of the calender.                    |
```

Comment:
Can you add `canvasTimeStart` and `canvasTimeEnd` into this table:

 canvasTimeStart: number, // denotes the start time in ms of the canvas timeline
  canvasTimeEnd: number, // denotes the end time in ms of the canvas timeline

Reply:
done